### PR TITLE
docs: Updates to main `README.md`

### PR DESCRIPTION
**Pull request by rogerthatdev**
This PR updates the main README:
- Reorganizes some of the information
- Updates command for running samples
- Adds step for compiling TypeScript samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,11 +56,19 @@ shows you how to use a variety of Google Cloud Platform features.
...
 
-[bookshelf_docs]: https://cloud.google.com/nodejs/getting-started/tutorial-app
-[bookshelf_code]: https://github.com/GoogleCloudPlatform/nodejs-getting-started
-
 ## Contributing
 
 Contributions welcome! See the [Contributing Guide][contrib].
 
-[contrib]: CONTRIBUTING.md
+[slack_badge]: https://img.shields.io/badge/slack-Google%20Cloud%20Platform-E01563.svg
```
Comment (pattishin) [inline README.md:63]: For now, leave this misc information as is in the readme and just have the ts build step added.

**Review by pattishin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rogerthatdev) [reply README.md]: Do you mean just the URLs I moved to the bottom of the page, or all of the changes I'm suggesting?

Comment (rogerthatdev) [reply README.md]: I moved the URLs back to where they were

**Review by pattishin — APPROVED** [review]

**Comment by pattishin** [comment]
Need to resolve the linting error, but otherwise all good.

**Pull request merged**